terraform/aws/analytical-platform-data-production/airflow/network.tf

```diff
@@ -137,6 +139,7 @@ resource "aws_cloudwatch_log_group" "airflow_dev_vpc_flow_log" {
   name              = "airflow-dev-vpc-flow-log"
   retention_in_days = 400
   skip_destroy      = true
+  kms_key_id        = aws_kms_key.cloudwatch.arn
```

Comment:
Don't currently encrypt these logs streams, agree we should, need to test whether this affects the ability of the Airflow UI to surface those logs to users.

Reply:
test in sandpit